pop-cli-for-appchains/guides/launch-a-chain/launch-a-chain-to-paseo.md

```diff
@@ -9,7 +9,7 @@ description: >-
 
 #### Paseo Local
 
-If you want to test onboarding a chain on your local machine you need to [launch Paseo](spinning-up-the-polkadot-network.md).
+If you want to test onboarding a chain on your local machine you need to [launch Paseo](./spinning-up-paseo.md).
```

Comment:
We even say "launch paseo" here. Another argument that spinning up might not be that clear.